packages/server/src/scim.ts

```diff
@@ -182,11 +219,6 @@ export class ScimServer implements DirectoryProvider {
 
         let objectId = (objectIdMatches && objectIdMatches[1]) || "";
 
-        // The id of groups is their name, so we base64 encode it, and need to decode here
-        if (url.pathname.startsWith("/Groups")) {
-            objectId = base64ToString(objectId);
-        }
```

Comment:
Using group names as resource ids is a bad idea, because...

```
      It MUST be a stable, non-reassignable identifier that does not change when the
      same resource is returned in subsequent requests.
```

(See https://datatracker.ietf.org/doc/html/rfc7643#section-3.1)